### PR TITLE
EVA-2403 - Check contigs can be translated to Genbank

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

File: tasks/eva_2357/test/test_populate_ids.py [hunk 1]
```diff
@@ -147,22 +149,26 @@ def test_populate_ids_fail(self, mock_get_mongo_uri_for_eva_profile):
...
-        self.assertCountEqual(variant['ids'], ['ss1'])
+        check_all_contigs(settings, databases_file, profile='localhost')
+
+    @patch('tasks.eva_2357.populate_ids.get_mongo_uri_for_eva_profile')
+    def test_check_all_contigs_raise(self, mock_get_mongo_uri_for_eva_profile):
+        logging.getLogger().setLevel(logging.DEBUG)
+        mock_get_mongo_uri_for_eva_profile.return_value = 'mongodb://127.0.0.1:27017'
+        settings = self.get_test_resource("settings.xml")
+        databases_file = self.get_test_resource("databases_errors.txt")
+        with pytest.raises(ValueError):
```
Comment (apriltuesday) [inline tasks/eva_2357/test/test_populate_ids.py:169]: If you do this you shouldn't need to import `pytest`...
```suggestion
        with self.assertRaises(ValueError):
```

File: tasks/eva_2406/tests/test_gather_release_species.py [hunk 2]
```diff
@@ -0,0 +1,32 @@
+#!/usr/bin/env python
```
Comment (apriltuesday) [inline tasks/eva_2406/tests/test_gather_release_species.py:1]: I think this and the next file must've slipped in from your other ticket!

**Review by apriltuesday — APPROVED** [review]

File: tasks/eva_2357/test/test_populate_ids.py (hunk 1 above)
Comment (tcezard) [reply tasks/eva_2357/test/test_populate_ids.py]: Done

File: tasks/eva_2406/tests/test_gather_release_species.py (hunk 2 above)
Comment (tcezard) [reply tasks/eva_2406/tests/test_gather_release_species.py]: Indeed 😊

**Review by andresfsilva — APPROVED** [review]

**Pull request merged**